### PR TITLE
Etienne/final rework

### PR DESCRIPTION
**Pull request by elafarge**
Some enhancement in the setup + updates with regard to automatic (and forced :@) TLS setup with Let's Encrypt. :feelsgood:

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -6,25 +6,27 @@ services:
...
+    # - "32469:32469"
+    # - "1900:1900/udp"
     environment:
     - PLEX_CLAIM="${PLEX_TOKEN}"
-    - ADVERTISE_IP="http://${IP_ADDRESS}:32400/"
-    - PLEX_UID="${UID}"
-    - PLEX_GID="${GID}"
+    - ADVERTISE_IP="https://plex.${DOMAIN_NAME}/"
+    - "PLEX_UID=${USER_ID}"
+    - "PLEX_GID=${GROUP_ID}"
```
Comment (hkaj) [inline docker-compose.yaml]: not sure about the quotes at the beginning

File: docker-compose.yaml [hunk 2]
```diff
@@ -38,16 +40,18 @@ services:
     - "51413:51413"
     - "51413:51413/udp"
     environment:
-    - PUID="${UID}"
-    - PGID="${GID}"
+    - "PUID=${USER_ID}"
+    - "PGID=${GROUP_ID}"
```
Comment (hkaj) [inline docker-compose.yaml]: ditto

File: docker-compose.yaml [hunk 3]
```diff
@@ -59,12 +63,14 @@ services:
     - ./media/sonarr/config:/config
     - ./media/transmission/downloads/:/downloads
     environment:
-    - PUID="${UID}"
-    - PGID="${GID}"
+    - "PUID=${USER_ID}"
+    - "PGID=${GROUP_ID}"
```
Comment (hkaj) [inline docker-compose.yaml]: ditto

File: docker-compose.yaml [hunk 4]
```diff
@@ -73,33 +79,52 @@ services:
...
     - "traefik.frontend.rule=Host:couchpotato.${DOMAIN_NAME}"
     - "traefik.port=5050"
+    - "traefik.docker.network=web"
     depends_on:
     - transmission
 
-  # uncomment to enable torznab as a T411 proxy for couchpotato and sonarr
+  # uncomment to enable torznab as a T411 proxy for sonarr
   # torznab's image will need to be built
   t411proxy:
```
Comment (hkaj) [inline docker-compose.yaml]: maybe comment out t411proxy by default? otherwise change the comment

File: docker-compose.yaml [hunk 5]
```diff
@@ -73,33 +79,52 @@ services:
...
     networks:
     - web
-    ports:
-    - "9876:9876"
     environment:
     - "USERNAME=${T411USERNAME}"
     - "PASSWORD=${T411PASSWORD}"
 
+  # Sonarr to... a bunch of public trackers!
+  jackett:
```
Comment (hkaj) [inline docker-compose.yaml]: cool addition :)

File: docker-compose.yaml [hunk 6]
```diff
@@ -73,33 +79,52 @@ services:
...
-    - /etc/localtime:/etc/localtime:ro
+    - /dev/rtc:/dev/rtc:ro
     - ./media/couchpotato/movies:/movies
     - ./media/couchpotato/config/:/config
     - ./media/transmission/downloads/:/downloads
     environment:
-    - PUID="${UID}"
-    - PGID="${GID}"
+    - "PUID=${USER_ID}"
+    - "PGID=${GROUP_ID}"
```
Comment (hkaj) [inline docker-compose.yaml]: ditto

File: docker-compose.yaml (hunk 1 above)
Comment (elafarge) [inline docker-compose.yaml]: Well it works but - God bless you - I'll put my quotes in the right places :)

File: docker-compose.yaml (hunk 2 above)
Comment (elafarge) [inline docker-compose.yaml]: :+1:

File: docker-compose.yaml (hunk 3 above)
Comment (elafarge) [inline docker-compose.yaml]: :+1:

File: docker-compose.yaml (hunk 6 above)
Comment (elafarge) [inline docker-compose.yaml]: :+1:

File: docker-compose.yaml (hunk 4 above)
Comment (elafarge) [inline docker-compose.yaml]: I think it's something we should prompt our user for in the future setup script but yeah, I'll comment it out.

**Pull request merged**